packages/next-app/src/app/pool/[poolId]/MetadataAttribute.tsx

```diff
@@ -1,54 +1,22 @@
+"use client";
+
 import {
   ArrowTopRightIcon,
   ChevronDownIcon,
   Pencil2Icon,
 } from "@radix-ui/react-icons";
 import cn from "classnames";
-import { TableHTMLAttributes } from "react";
+import { TableHTMLAttributes, useContext } from "react";
 
 import {
   Button,
   ImageDialog,
   PoolMetadataFormModal,
 } from "../../../components";
-
-const cellData: {
-  name: string;
-  type: string;
-  desc: string;
-  value: React.ReactNode;
-}[] = [
-  {
-    name: "Pool Address",
-    type: "address",
-    desc: "The address of the smart contract that implements the exchange pool",
-    value: "0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D",
-  },
-  {
-    name: "Pool link",
-    type: "URL",
-    desc: "The address of the smart contract that implements the exchange pool",
-    value: "https://github.com/",
-  },
-  {
-    name: "Pool Address",
-    type: "address",
-    desc: "The address of the smart contract that implements the exchange pool",
-    value: "0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D",
-  },
-  {
-    name: "Pool link",
-    type: "URL",
-    desc: "The address of the smart contract that implements the exchange pool",
-    value: "https://github.com/",
-  },
-  {
-    name: "Pool image",
-    type: "image",
-    desc: "balancer logo",
-    value: "https://s2.coinmarketcap.com/static/img/coins/200x200/5728.png",
-  },
-];
+import {
+  PoolMetadataAttribute,
+  PoolMetadataContext,
+} from "../../../contexts/PoolMetadataContext";
```

Comment:
minor thing but could you create an issue to add [path aliases](https://nextjs.org/docs/advanced-features/module-path-aliases) to the project?

Reply:
Yep, thanks 👍

Reply:
You can use the same convention adopted by Remix here:
```
# tsconfig.json
...
    "paths": {
      "~/*": ["./app/*"]
    },
...
```